files/keepalived_notifications.sh

```diff
@@ -13,11 +13,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-TYPE=$1
-NAME=$2
+#TYPE=$1
+#NAME=$2
 NOW=`date "+%Y-%m-%d %H:%M:%S"`
 NEWSTATE=$3
-OLDSTATE=$(cat /var/run/keepalived.state)
+#OLDSTATE=$(cat /var/run/keepalived.state)
```

Comment:
You don't need to add this again. I am not sure why it's still there. This was just an example anyway, so feel free to remove the line altogether.

Reply:
Absolutely, I've removed that line now.